tests/system/providers/google/cloud/datapipelines/google-datapipeline.py

```diff
@@ -0,0 +1,118 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG for testing Google DataPipelines Create Data Pipeline Operator.
+"""
+from __future__ import annotations
+
+import os
+from datetime import datetime
+from pathlib import Path
+
+from airflow import models
+from airflow.providers.google.cloud.operators.datapipeline import (
+    CreateDataPipelineOperator,
+)
+from airflow.providers.google.cloud.operators.gcs import GCSCreateBucketOperator, GCSDeleteBucketOperator
+from airflow.providers.google.cloud.transfers.local_to_gcs import LocalFilesystemToGCSOperator
+from airflow.utils.trigger_rule import TriggerRule
+
+DAG_ID = "google-datapipeline"
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+GCP_PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
+GCP_LOCATION = os.environ.get("location", "us-central1")
+
+PIPELINE_NAME = "defualt-pipeline-name"
+PIPELINE_TYPE = "PIPELINE_TYPE_BATCH"
+
+BUCKET_NAME = f"bucket_{DAG_ID}_{ENV_ID}"
+
+FILE_NAME = "kinglear.txt"
+TEMPLATE_FILE = "word-count.json"
+DATAPIPELINES_JOB_NAME = "test-job-name"
+TEMP_LOCATION = f"gs://{BUCKET_NAME}/temp"
+
+GCS_PATH = f"gs://{BUCKET_NAME}/templates/{TEMPLATE_FILE}"
+INPUT_FILE = f"gs://{BUCKET_NAME}/examples/{FILE_NAME}"
+OUTPUT = f"gs://{BUCKET_NAME}/results/hello"
+
+FILE_LOCAL_PATH = str(Path(__file__).parent / "resources" / FILE_NAME)
+TEMPLATE_LOCAL_PATH = str(Path(__file__).parent / "resources" / TEMPLATE_FILE)
```

Comment:
I think this files are missing in this PR, aren't they?

Reply:
@ahidalgob Updated